modular_skyrat/modules/opposing_force/code/equipment/spells.dm

```diff
@@ -36,12 +36,12 @@
 	name = "Knock"
 	item_type = /obj/item/book/granter/spell/knock
 	description = "This spell opens nearby doors and closets."
-
+/* //Requested to be removed by host for being too LRP
```

Comment:
Why keep it commented in?

Reply:
I'd say for consistency to show that any spellbooks weren't missed, but... eh, that's what git blame is for